### ISSUE TITLE
I want to know what data Tranformations used while training 

### ISSUE THREAD

**Issue by Dheeru66k** [DrSleep/multi-task-refinenet#11]
Hi I wanted to know about your dataloader part for NYUD dataset
I am Normalizing and converting to tensor, in my ToTensor() function i am reading the data and converting them to float.
![Screenshot from 2023-02-08 12-16-58](https://user-images.githubusercontent.com/64223954/217515101-59f7be2d-7a07-488f-8a16-2f9b5d8f39ed.png)

is this correct, if possible please the dataloaders used in this papers

**Comment by DrSleep** [comment]
See here: https://github.com/DrSleep/multi-task-refinenet#more-to-come

In particular, https://github.com/DrSleep/DenseTorch/blob/master/examples/multitask/train.py#L20-L24

**Comment by Dheeru66k** [comment]
Hi, Thanks for the reply
I have referred to the "DenseTorch" repo and tried running MTL model with depth ans Seg,its actually working and after  600 epocs the loss and MeanIoU are became stable with no further improvement as below.(Used your NYUD dataset)
![image](https://user-images.githubusercontent.com/64223954/218257273-96bad255-ccd1-45b4-abbf-15f86df60065.png)


Issue 2:
If i want to use Same encoder and Decoder used in Multitask model(mobilenet, Lightweigh Refinenet) for training individual heads, for example if i want to train only Depth, what exactly i need to change 


![Screenshot from 2023-02-11 13-14-20](https://user-images.githubusercontent.com/64223954/218257428-20333968-6c6c-4794-af08-4148efdc3e07.png)

and what parameters to give for depth, if i use Mobilenet and refinenent, instead of xception65, DLv3plus as below(showing side by side Multi(left) and Single tasks(right))
![Screenshot from 2023-02-11 13-16-09](https://user-images.githubusercontent.com/64223954/218257580-84cbdd89-2416-4907-aa74-1ce9a307aae0.png)

**Comment by DrSleep** [comment]
If you want to change the number of tasks, you need to adapt accordingly `masks_names`, `criterions`, `loss_coeffs`, `num_classes` and `metrics`. You can compare the configs between the single-task training example and the multi-task training example to confirm that. 

For exact hyperparameters, it is better to refer to the original paper(s), iirc DenseTorch default values differ from those in the paper.
